src/Servers/HttpSys/samples/TlsFeaturesObserve/TlsFeaturesObserve.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore" />
```

Comment:
What's this for?

Reply:
for minimal API usage. its only a sample, so should not be a big problem?